LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Effect_Brawler.uc

```diff
@@ -0,0 +1,30 @@
+//--------------------------------------------------------------------------------------- 
+//  FILE:    X2Effect_Brawler
+//  AUTHOR:  Grobobobo
+//  PURPOSE: Sets up the DR bonus for brawler
+//---------------------------------------------------------------------------------------
+class X2Effect_Brawler extends X2Effect_Persistent config (LW_SoldierSkills);
+
+var config float BRAWLER_DR_PCT;
+var config int BRAWLER_MAX_TILES;
+
+
+function int GetDefendingDamageModifier(XComGameState_Effect EffectState, XComGameState_Unit Attacker, Damageable TargetDamageable, XComGameState_Ability AbilityState, const out EffectAppliedData AppliedData, const int CurrentDamage, X2Effect_ApplyWeaponDamage WeaponDamageEffect, optional XComGameState NewGameState)
+{
+	local int   Tiles;
+
+	Tiles = Attacker.TileDistanceBetween(XComGameState_Unit(TargetDamageable));       
+	if (Tiles <= default.BRAWLER_MAX_TILES - 1)
+	{
+        return -currentdamage * default.BRAWLER_DR_PCT / 100;
```

Comment:
```suggestion
        return -CurrentDamage * default.BRAWLER_DR_PCT / 100;
```

Reply:
hold on, how did that one get past the compilator? It also worked on testing and all,

Reply:
UnrealScript is case insensitive, so it compiles fine